opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -180,7 +180,7 @@ where
         // Visit fields.
         event.record(&mut visitor);
 
-        #[cfg(feature = "experimental_use_tracing_span_context")]
+        /*#[cfg(feature = "experimental_use_tracing_span_context")]
```

Comment:
It might be obvious to everyone else, but i'm wondering what's with these two features?

Reply:
This will be enabled through a patch release for appender-tracing once the new version of tracing-opentelemetry is available. The conflict arises because appender-tracing and tracing-opentelemetry depend on different versions of the opentelemetry-sdk crate, which includes breaking changes between versions.